src/action.ts

```diff
@@ -66,12 +66,13 @@ enum ExecutionMode {
   SynthOnly = "synth-only",
   PlanOnly = "plan-only",
   AutoApproveApply = "auto-approve-apply",
+  AutoApproveDestroy = "auto-approve-destroy",
 }
 
 async function execute(
   cdktfCommand: string,
-  reportSuccess: (output: string, runUrl?: string) => Promise<void>,
-  reportFailure: (
+  reportSuccess?: (output: string, runUrl?: string) => Promise<void>,
```

Comment:
Let's keep these required and report back to the user. One can still disable it by setting commentOnPR to false

Reply:
When you say "report back to the user", what would you like that change to look like. 
Leaving a PR comment like the others? Or logging a message in the action with a console log?

I didn't think you'd want to encourage PRs that teardown infra, but I suppose having the option to leave a comment afterward doesn't make that any more or less likely to happy.